themes/vue/layout/partials/ecosystem_dropdown.ejs

```diff
@@ -12,6 +12,7 @@
       <ul>
         <li><a href="https://github.com/vuejs/vue-devtools" class="nav-link" target="_blank">Devtools</a></li>
         <li><a href="https://vuejs-templates.github.io/webpack" class="nav-link" target="_blank">Webpack Template</a></li>
```

Comment:
I would even say we should remove the webpack template from the dropdown.

We won'T put much energy in its maintanance now that vue-cli 3 is stable, so while it will continue to be available, we should not put it on the same "level" as the CLIU here.

Can you remove the link?

Reply:
That makes sense. Done!